### ISSUE TITLE
Artwork missing 

### ISSUE THREAD

**Issue by Bonovox78** [PopcornTimeTV/PopcornTimeTV#859]
Hi,

I’m noticing artwork for a lot of the movies is missing.

Can this be fixed?

**Comment by Midifreakz** [comment]
This seems to happen from time to time. It usually fixes itself within a few days.

**Comment by suncreations** [comment]
Same as. It happened out of the blue. Not sure if this is related to Trakt.tv. Does it actually sorts itself out?

**Comment by Midifreakz** [comment]
Series and favorites are working on my end, movies are out.  As I said above, this hapens from time to time :(

**Comment by suncreations** [comment]
Yeah same here, all artwork showing except 99% of movies. Thanks for confirming though, at least I know that they’ll show up again.

**Comment by iAYMANi** [comment]
Yeah same here
